vignettes/cellxgene_basic.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "Basic CELLxGENE workflow"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Wetlab Module}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+- CELLxGENE
+- What's in this tutorial?
+  - Connect to CELLxGENE
+  - Download a dataset
+  - Convert to Seurat
+  - Perform simple analysis
+  - Save and upload results
+  - Save and upload report (?)
+  
+# Connect to a LaminDB instance
+
+- Connect to CELLxGENE instance
+- Show instance object
+
+# Downloading a dataset
+
+- Show Artifact registry
+- Go to CELLxGENE Lamin website
+- Find a dataset, get the ID
+- Cache and load the object locally
+
+# Convert to Seurat
+
+- CELLxGENE stores AnnData, we want Seurat
+- Do the conversion
+
+# Analysis
+
+- Follow the Seurat tutorial to calculate marker genes
+- Save results as a text file
+
+# Add results
+
+- Add the results as an artifact
+- Render and add report (?)
```

Comment:
We could use a `.ipynb` to run code but I'm not sure we could add it to the current website (if that's the place). Also, it's not commonly used for R analysis. There is also Quarto which can be used for R and Python.

I'm not sure if we could run `lamin save` inside the same document or if that would break. I don't know enough about how it all works to say what the best approach might be.

Reply:
> Also, it's not commonly used for R analysis.

Then this should be the last resort. We should use what's commonly used.

> I don't know enough about how it all works to say what the best approach might be.

We should think about the best approach for an R user, not about what's easiest for lamin. I'll try to call you.

Reply:
@lazappi and I just went on a call and hashed out the UX.

1. The user exports their current `.qmd` or `.rmd` session as an html with the same filename but the `.html` suffix
2. The user runs `lamin save myanalysis.qmd` on the terminal

If the user forgets to export a `.html` the CLI will raise an error and remind the user to do that manually.